src/main/java/uk/gov/ida/matchingserviceadapter/rest/matchingservice/MdsValueOnlyDto.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.ida.matchingserviceadapter.rest.matchingservice;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.joda.time.DateTime;
+
+@JsonSerialize(include= JsonSerialize.Inclusion.NON_NULL)
```

Comment:
Class is not in use. (Might as well because I have no idea what it's trying to do based on the name...)